### PR TITLE
Report portal Integration

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
https://github.com/saikrishna321/AppiumTestDistribution/issues/350

### CODE REVIEW

**Comment by saikrishna321** [comment]
@SrinivasanTarget why is the device info repeated 

screen shot 2018-07-15 at 9 44 54 pm

File: src/main/java/com/appium/utils/ScreenShotManager.java [hunk 1]
```diff
@@ -150,8 +152,14 @@ private void screenShotAndFrame(int status,
...
             } else {
-                FileUtils.copyFile(scrFile, new File(System.getProperty("user.dir")
-                        + FileLocations.OUTPUT_DIRECTORY + getCapturedScreen().trim()));
+                String capturedScreenshotPath = System.getProperty("user.dir")
+                        + FileLocations.OUTPUT_DIRECTORY + getCapturedScreen().trim();
+                FileUtils.copyFile(scrFile, new File(capturedScreenshotPath));
+                if (new File(capturedScreenshotPath).exists()) {
+                    ReportPortal.emitLog(screenShotName, "Trace", new Date(), new File(capturedScreenshotPath));
+                } else {
+                    throw new FileNotFoundException();
```
Comment (saikrishna321) [inline src/main/java/com/appium/utils/ScreenShotManager.java]: Why are we throwing an exception?

**Comment by SrinivasanTarget** [comment]
Codacy need to wake up and pick latest commit :p

**Comment by saikrishna321** [comment]
screen shot 2018-07-19 at 10 25 20 pm

**Comment by SrinivasanTarget** [comment]
PR is not ready to merge. 45b0488bd5eefeba4e767782593c9c7a8e853a6c has broken normal screenshots func.

**Comment by saikrishna321** [comment]
@SrinivasanTarget has this broken the failure screenshot ?

**Comment by SrinivasanTarget** [comment]
yeah @saikrishna321 I will test it throughly though tomorrow.

**Comment by saikrishna321** [comment]
Sorry my bad i broke it

**Pull request merged**